### DISCUSSION TITLE
[Feature Request] Interface for determining if HDF5 was built with support for parallel enabled

### DISCUSSION THREAD

**Discussion by jhendersonHDF** [HDFGroup/hdf5#3403] [Ideas]
A user made a feature request to add an interface that would return whether HDF5 was built with parallel/MPI support, similar to H5is_library_threadsafe. Applications can currently check this through the use of `#ifdef H5_HAVE_PARALLEL`, though there may be some use in having an API routine that returns this information.

**Comment by simonbyrne** [comment]
This would be very helpful for the Julia bindings ([HDF5.jl](https://github.com/JuliaIO/HDF5.jl/)) as we have no way of querying the header macros (we call directly into the library).
